content/06.methods.md

```diff
@@ -188,56 +179,57 @@ CNVs called significant by Control-FREEC (p-value < 0.01) and Manta calls that p
 
-We first removed genes with no expression from the RSEM abundance data using a cut-off of FPKM > 0 in at least 1 sample across the PBTA cohort.
-We computed the mean FPKM across all samples per gene and for each multi-mapped gene symbol, we chose the Ensembl identifier corresponding to the maximum mean FPKM with the goal of choosing the identifier that best represented the expression of the gene.
-After collapsing gene identifiers, there were a total of 46,400 unique expressed genes in the poly-A dataset and a total of 53,011 unique expressed genes remaining in the stranded dataset.
-More detail can be found in the [`collapse-rnaseq` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/collapse-rnaseq).
+We first removed genes with no expression from the RSEM abundance data by requiring an FPKM > 0 in at least 1 sample across the PBTA cohort.
+We computed the mean FPKM across all samples per gene. 
+For each multi-mapped gene symbol, we chose the Ensembl identifier corresponding to the maximum mean FPKM with the goal of choosing the identifier that best represented the expression of the gene.
+After collapsing gene identifiers, there were a total of 46,400 uniquely expressed genes in the poly-A dataset and a total of 53,011 uniquely expressed genes remaining in the stranded dataset.
+Additional details can be found in the [`collapse-rnaseq` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/collapse-rnaseq).
 
 ##### Gene fusion detection
 
-We set up [Arriba v1.1.0](https://github.com/suhrig/arriba/) and STAR-Fusion 1.5.0 [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
-For both of these tools, we used aligned BAM and chimeric SAM files from STAR as inputs and GRCh38_gencode_v27 GTF for gene annotation.
-We ran STAR-Fusion with default parameters and annotated all fusion calls with GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz provided in the STAR-fusion release.
-For Arriba, we used a blacklist file (blacklist_hg38_GRCh38_2018-11-04.tsv.gz) from the Arriba release to remove recurrent fusion artifacts and transcripts present in healthy tissue.
-We also provided Arriba with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
-We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) on Arriba fusion calls in order to harmonize annotations with those of STAR-Fusion.
-The RNA expression and fusion workflows can be found in the [OpenPBTA workflows GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_RNAseq_workflow.cwl).
-The FusionAnnotator workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/FusionAnnotator).
+We set up Arriba v1.1.0 [@doi:10.1101/gr.257246.119] and STAR-Fusion 1.5.0 [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
```

Comment:
`v` or no `v` for both versions (unless versions themselves really have diff names like that!)